syntax = "proto3";

package proto;

import "messages.proto";

service Indicadores {
  rpc ObtemIndicadores (PpgRequest) returns (PpgResponse);
}
